sample.py

```diff
@@ -0,0 +1,19 @@
+
+
+import sys
+import os
+def fib():
+    x = 0
```

Comment:
__[flake8]__
* [F841] local variable 'x' is assigned to but never used

<!-- Comment by [SideCI](https://sideci.com) -->